omod/src/main/webapp/pages/allergy.gsp

```diff
@@ -60,11 +60,11 @@ ${ ui.includeFragment("allergyui", "removeAllergyDialog") }
             <div id="allergens">
 
                 <div id="types" class="button-group horizontal">
-                    <% allergenTypes.each { category -> %>
+                    <% allergenTypes.each { category ->  if (category.name()!="ENVIRONMENT") { %>
```

Comment:
So if my database has an allergy of type ENVIRONMENT, will it show up under the OTHER tab? If you are to say yes, have you actually tested and confirmed it?